src/config/android/index.js

```diff
@@ -38,6 +38,11 @@ exports.projectConfig = function projectConfigAndroid(folder, userConfig) {
     userConfig.mainActivityPath || `src/main/java/${packageFolder}/MainActivity.java`
   );
 
+  const stringsPath = path.join(
+    sourceDir,
+    userConfig.stringsPath || 'src/main/res/values/strings.xml'
```

Comment:
What means "custom strings.xml"? Strings.xml should always points to the same file